server/services/rulesEngineService.js

```diff
@@ -0,0 +1,55 @@
+const { runEngine, rules } = require('@neilbmclaughlin/rules-engine-test')
+
+const rulesMap = {
+  1: rules.perimeter,
+  2: rules.adjustedPerimeter,
+  3: rules.tolerancePerimeter,
+  4: rules.noActionsInTimePeriod,
+  5: rules.notSSSI,
+  6: rules.pondlessArea,
+  7: rules.area
+}
+
+module.exports = {
+  doEligibilityRun: async function (rules, parcels, passedFacts, successFunc) {
+    const eligibilityRules = rules.filter(rule => rule.type === 'eligibility')
+    await this.doFullRun(eligibilityRules, parcels, passedFacts, successFunc)
+  },
+
+  doFullRun: async function (requestedRules, parcels, passedFacts, successFunc, returnFacts) {
+    try {
+      if (parcels.length !== 1) {
+        throw Error('rulesEngineService.doFullRun requires exactly 1 parcel')
+      }
+
+      const parcel = parcels[0]
```

Comment:
You could just do `const [parcel] = parcels` here

Reply:
Yep, done that.